### DISCUSSION TITLE
Footer vs. color scheme

### DISCUSSION THREAD

**Discussion by AFulgens** [squidfunk/mkdocs-material#6545] [Q&A] [answered]
As per the [change request guidelines](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/), instead of directly opening an issue, I am opening a discussion.

Currently colors are customizable but not for the footer. See e.g., renderings of partials `copyright` and `social` in the official docu. The footer is fixed to the built-in black-and-white color-scheme, even though the docu has blue as primary color when seeing it in the "light" version.

My idea would be that the footer should have the same color (and changing behavior) as the header, i.e., it should respect `palette[*].primary` and `palette[*].accent` and the currently active `scheme`. Is this worth creating an issue for it?

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @AFulgens,
a new issue could be opened up with a reasoning of "fix integrity of the color palette scheme", but I'm no graphic designer to say if that's really necessary. The footer uses `--md-footer...` CSS variables, so I believe the current way is intentional to create some sort of contrast. 
Also users can always make changes to the styling using extra CSS:
https://squidfunk.github.io/mkdocs-material/customization/#additional-css
https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#customization
https://github.com/squidfunk/mkdocs-material/blob/master/src/templates/assets/stylesheets/main/_colors.scss

```css
:root > * {
  --md-footer-bg-color: var(--md-primary-fg-color);
}
```

**Reply by squidfunk** [reply]
The footer is deliberately styled differently and can be customized with CSS, so this is the correct answer.

**Reply by AFulgens** [reply]
Sadly, it's not that easy to make it consistent with the rest of the page, because the naming is not in-line with the rest of the color palette‚ and for me for some reason (on Windows), if I switch to the `var(--md-...)` values, the footer still does not follow changing the palette. My CSS-fu is not that good that I can investigate this further, I will just accept this as a design decision of the framework. Thanks for the answers!

**Reply by kamilkrzyskow** [reply]
Weird worked for me on Firefox, didn't check Chrome etc. 
You can also check the web dev tools and see the hex value and set it to that instead of another var.
If the site is public I could check it out too.
